### DISCUSSION TITLE
[Geeral] <Title> check my wallet

### DISCUSSION THREAD

**Discussion by vahidbnb** [zkSync-Community-Hub/zksync-developers#584] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

0xEC03CeE1E6E218942C5E7f5B92dDB3a6e08026Fc

Pls check 

+120 tx in era 
+100 tx in lite before snapshat 2023
+50 gas fee
+50 smart contract 
+75 tx in eth mainnet 
i have 1 wallet and not eligible

### Additional Details

0xEC03CeE1E6E218942C5E7f5B92dDB3a6e08026Fc

Pls check 

+120 tx in era 
+100 tx in lite before snapshat 2023
+50 gas fee
+50 smart contract 
+75 tx in eth mainnet 
i have 1 wallet and not eligible

**Comment by 0xsisyfos** [comment, marked as answer]
Hello,

Thank you for reaching out.

The ZK Token FAQ provides additional details on eligibility criteria: https://docs.zknation.io/zk-token/zk-token-faq

In particular, question 3 ("I met some of the eligibility criteria, but did not get an allocation, why not?") answers your question directly.
